src/FixedRankMatrices.jl

```diff
@@ -0,0 +1,193 @@
+using LinearAlgebra: diag, Diagonal, svd, SVD, rank, dot
+import Base: \, /, +, -, *
+@doc doc"""
+    FixedRankMatrices{M,N,K} <: Manifold
```

Comment:
Can you document what the default metric choice is?

Reply:
Yes, of course.